src/oneclick/discovery/discoveryReport.py

```diff
@@ -84,24 +92,36 @@ def run(cls,config:Config):
             lang_list = list(nsuport['LANGUAGE'])
             lang_list[-1]=f'and {lang_list[-1]}'
             bsup_lang = ', '.join(lang_list)
-            doc.add_paragraph(f"The remaining {len(nsuport)} are irrelevant and will not be analyzed, {bsup_lang}.",style='List Bullet 2')
+            doc.add_paragraph(f"The remaining {len(nsuport)} technologies/extensions are not relevant and will not be analyzed. These include {bsup_lang}.",style='List Bullet')
 
             # in scope code base
-            asuport = after_df[before_df['APPLICABLE']==True]
-            total = int(asuport['CODE'].sum())
-            total, unit = convert_LOC(total)
-            doc.add_paragraph(f'After removing all Sample, Test and other non production related code there is ({total} {unit}) in scope for this project',style='List Bullet 2')
-
-            line = []
-            for index, row in sql_df.iterrows():
-                line.append(f"{row['Unique']} {row['Catagory']}")
-                if row['Catagory'] == 'SQL files':
-                    line[-1]=f'in {line[-1]}'
-            doc.add_paragraph(f"The delivery also inclues, {', '.join(line)}.",style='List Bullet')
-
-            doc.add_paragraph("Here is a high-level LOC per core techno of the in-scope code taken using CLOC utility",style='List Bullet')
+            # asuport = after_df[before_df['APPLICABLE']==True]
+            # total = int(asuport['CODE'].sum())
+            # total, unit = convert_LOC(total)
+            #doc.add_paragraph(f'After removing all Sample, Test and other non production related code there is ({total} {unit}) in scope for this project',style='List Bullet 2')
+            #print(sql_df)
+            tables=sql_df['Total'][0]
+            views=sql_df['Total'][4]
+            procedures=sql_df['Total'][1]
+            doc.add_paragraph(f'The database contains a total of  {tables} tables, {views} views and {procedures} stored procedures/functions.',style='List Bullet')
```

Comment:
Do not include sql items with zero totals
The SQL report includes Tables, Procedures, Functions, Triggers and Views, include all in the bullet item